tests/tcl/tests/unit/type/list-2.tcl

```diff
@@ -1,70 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-# Copyright (c) 2006-2020, Salvatore Sanfilippo
-# See bundled license file licenses/LICENSE.redis for details.
-
-# This file is copied and modified from the Redis project,
-# which started out as: https://github.com/redis/redis/blob/dbcc0a8/tests/unit/type/list-2.tcl
-
-start_server {
-    tags {"list"}
-    overrides {
-        "list-max-ziplist-size" 4
-    }
```

Comment:
Interesting. It seems we don't populate this config in the new test case but it still passes.

Reply:
To populate config, see:

https://github.com/apache/incubator-kvrocks/blob/3f0fb401030332aff76a4d2638887cbf0de240fc/tests/gocase/unit/auth/auth_test.go#L45-L47

Reply:
fixed